src/test/java/core/basesyntax/service/impl/FruitTransactionProcessorImplTest.java

```diff
@@ -0,0 +1,90 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.fail;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.FruitTransactionProcessor;
+import core.basesyntax.strategy.OperationCalculator;
+import core.basesyntax.strategy.OperationStrategy;
+import core.basesyntax.strategy.impl.CreditOperationHandler;
+import core.basesyntax.strategy.impl.DebitOperationHandler;
+import core.basesyntax.strategy.impl.OperationStrategyImpl;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FruitTransactionProcessorImplTest {
+    private static FruitTransactionProcessor fruitTransactionProcessor;
+
+    @BeforeClass
+    public static void beforeAll() {
+        Map<FruitTransaction.Operation, OperationCalculator> operationsMap = Map.of(
+                FruitTransaction.Operation.BALANCE, new CreditOperationHandler(),
+                FruitTransaction.Operation.SUPPLY, new CreditOperationHandler(),
+                FruitTransaction.Operation.PURCHASE, new DebitOperationHandler(),
+                FruitTransaction.Operation.RETURN, new CreditOperationHandler()
+        );
+        OperationStrategy operationStrategy = new OperationStrategyImpl(operationsMap);
+        fruitTransactionProcessor = new FruitTransactionProcessorImpl(operationStrategy);
+    }
+
+    @Test
+    public void handleData_Ok() {
```

Comment:
I wouldn't say I like this method name, cause it doesn't mean for me what this test is testing, but maybe it is ok, so it is up to you in its naming :)